packages/lib/BaseModel.ts

```diff
@@ -80,6 +80,8 @@ class BaseModel {
 		['TYPE_COMMAND', ModelType.Command],
 	];
 
+	private static uuidGenerator: ()=> string = uuid.create;
```

Comment:
This change was added so we can assign a `uuidGenerator` to Models and create reproducible snapshots.